docs/rule/groovy_library.soy

```diff
@@ -0,0 +1,168 @@
+{namespace groovy_library}
+
+/***/
+{template .soyweb}
+  {call buck.page}
+    {param title: 'groovy_library()' /}
+    {param prettify: true /}
+    {param description}
+      A rule that is used to define a set of Groovy files that can be compiled together.
+
+      It can also be used to cross compile a set of Groovy and Java files.
+    {/param}
+    {param content}
+
+{call buck.rule}
+{param status: 'FROZEN' /}
```

Comment:
In retrospect, that is well named enough that it should have actually triggered a response in my brain. Good catch, thanks.